src/slack_message_formatter.ts

```diff
@@ -1,11 +1,24 @@
 import { HistoricalMessage } from './types';
 import logger from './logger';
 
-const formatMessageBlock = (msg: string, formatchar: string) => {
+const formatMessageBlock = (
+  msg: string,
+  attachments: string[] | null | undefined,
+  formatchar: string
+) => {
   return msg
     .split('\n')
     .map((x) => (x ? formatchar + x + formatchar : x))
     .map((x) => '>' + x)
+    .concat(
+      attachments?.length
+        ? [
+            `*Attachments:* ${attachments
+              .map((url, i) => `<${url}|Attachment ${i + 1}>`)
+              .join(' ')}`,
+          ]
+        : []
+    )
```

Comment:
I considered displaying the message history using blocks so I could intersperse buttons for attachments as needed, but it ended up taking more vertical space and making things unnecessarily complicated for an edge case, so I decided to just stick with plain ol' links in this case. I imagine the need to open attachments from before routing will be rare.

<img width="398" alt="Screen Shot 2020-10-28 at 12 51 15 AM" src="https://user-images.githubusercontent.com/179327/97411855-0a43cb00-18be-11eb-9baa-80d6486a6ecf.png">

Reply:
This is absolutely fine! Thanks for thinking of this